hack/bazel-push-images.sh

```diff
@@ -29,31 +29,39 @@ default_targets="
     virt-controller
     virt-handler
     virt-launcher
-    virt-exportserver
-    virt-exportproxy
-    conformance
-    libguestfs-tools
-    pr-helper
-    example-hook-sidecar
-    example-disk-mutation-hook-sidecar
-    example-cloudinit-hook-sidecar
     alpine-container-disk-demo
-    cirros-container-disk-demo
-    cirros-custom-container-disk-demo
-    virtio-container-disk
-    alpine-ext-kernel-boot-demo
     fedora-with-test-tooling-container-disk
-    alpine-with-test-tooling-container-disk
-    fedora-realtime-container-disk
-    disks-images-provider
-    nfs-server
     vm-killer
-    winrmcli
     sidecar-shim
-    network-slirp-binding
-    network-passt-binding
 "
 
+case ${ARCHITECTURE} in
```

Comment:
I am not sure I understand this change.

Reply:
Since aarch64/x86_64 share the image list, the default values are already all the images we support currently for s390x. Therefore this case simply does nothing for s390x and adds the original images otherwise.
I choose a select instead of an if here because this makes it easier to expand for other arches if needed and reflects the use of select statements in the go code for arch specific cases as well.

Reply:
Ok, so s390x doesn't support this, thanks I was not synced...